### PR TITLE
add empty project for species diversity microservice

### PR DESCRIPTION
**Pull request by Trjegul84**

### CODE REVIEW

File: services/diversity/config/default.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "db": {
+      "host": "localhost",
+      "port": 5432,
+      "user": "user",
+      "password": "password",
+      "database": "database"
+    },
+    "server" : {
+      "port": 6000,
```
Comment (erikasv) [inline services/diversity/config/default.json:10]: @cazapatamar had problems with that port number, I wold suggest to use 4003 to keep it simpler and closer to the other services

File: services/diversity/src/util/errorHandler.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    logger.info(`[${req.id()}] received - ${req.href()}`);
+    try {
+      await callback(req, res, next);
+      logger.info(`[${req.id()}] finished - ${req.href()}`);
+    } catch (e) {
+      const code = typeof e.code === 'number' ? e.code : 500;
+      logger.error(e.stack || e.Error || e.message || e);
+      res.send(code, {
+        code,
+        userMsg: e.message || e.userMsg || 'There was an internal error',
```
Comment (erikasv) [inline services/diversity/src/util/errorHandler.js:20]: I believe this order was changed in connectivity service, please put `e.userMsg` before `e.message` and also change it in the other services.

**Review by erikasv — CHANGES_REQUESTED** [review]

File: services/diversity/config/default.json (hunk 1 above)
Comment (cazapatamar) [reply services/diversity/config/default.json:10]: Agree

Comment (Trjegul84) [reply services/diversity/config/default.json]: Changed!

File: services/diversity/src/util/errorHandler.js (hunk 2 above)
Comment (Trjegul84) [reply services/diversity/src/util/errorHandler.js]: Ok. Done

**Review by erikasv — APPROVED** [review]

**Pull request merged**